src/Microsoft.AspNetCore.Server.HttpSys/AuthenticationHandler.cs

```diff
@@ -112,14 +112,14 @@ public Task SignOutAsync(SignOutContext context)
             return TaskCache.CompletedTask;
```

Comment:
These comments about !Accepted aren't true anymore. SignIn should throw NotSupportedException. SignOut is something we're discussing elsewhere.